6.bulk_Morphology_Elastic_Network/4.model_performance/scripts/2.model_performance_visualization.r

```diff
@@ -0,0 +1,278 @@
+suppressPackageStartupMessages(suppressWarnings(library(ggplot2)))
+suppressPackageStartupMessages(suppressWarnings(library(dplyr)))
+suppressPackageStartupMessages(suppressWarnings(library(ComplexHeatmap)))
+suppressPackageStartupMessages(suppressWarnings(library(reshape2)))
+suppressPackageStartupMessages(suppressWarnings(library(ggplotify)))
+suppressPackageStartupMessages(suppressWarnings(library(RColorBrewer)))
+suppressPackageStartupMessages(suppressWarnings(library(circlize)))
+
+# set cell type
+cell_type = "PBMC"
```

Comment:
Depending on how this might effect things, consider being consistent with this assignment using `<-` instead of `=`.

Reply:
Change made